x-pack/metricbeat/module/aws/cloudwatch/cloudwatch.go

```diff
@@ -189,14 +192,49 @@ func (m *MetricSet) Fetch(report mb.ReporterV2) error {
 
 		for namespace, namespaceDetails := range namespaceDetailTotal {
 			m.logger.Debugf("Collected metrics from namespace %s", namespace)
-
 			// filter listMetricsOutput by detailed configuration per each namespace
 			filteredMetricWithStatsTotal := filterListMetricsOutput(listMetricsOutput, namespace, namespaceDetails)
 
 			// get resource type filters and tags filters for each namespace
 			resourceTypeTagFilters := constructTagsFilters(namespaceDetails)
 
-			eventsWithIdentifier, err := m.createEvents(svcCloudwatch, svcResourceAPI, filteredMetricWithStatsTotal, resourceTypeTagFilters, regionName, startTime, endTime)
+			//Check whether namespace is APIGW
+			checkns := "AWS/ApiGateway"
+			checkresource_type := "apigateway:restapis"
```

Comment:
nit - I think these should be constants as they are AWS constants. And we could derive their lowercase var to avoid lower string generation in this for loop :)

Reply:
Made them as const in https://github.com/orgs/elastic/projects/1228/views/18

Reply:
Nice :) 

Further, we could derive their lowercase var so we don't have to use  for example `strings.ToLower(checkresource_type)` in loops (which is as costly as ToLower generate a new string)

Reply:
Done in [7bd7324](https://github.com/elastic/beats/pull/40755/commits/7bd732430804135067685728fbf0e669f4abd278)